languages/de-DE/Buttons.multids

```diff
@@ -30,12 +30,22 @@ ExportTiddler/Caption: Exportieren
 ExportTiddler/Hint: Diesen Tiddler Exportieren
 ExportTiddlers/Caption: Mehrere Exportieren
 ExportTiddlers/Hint: Mehrere Tiddler Exportieren
+Fold/Caption: Ausblenden Textbereich
```

Comment:
Then I would turn it around and call it "Textbereich Ausblenden" ...actually, why would all these Strings be Capitalized? Would it not be better to have them in their natural version? It may not look so pretty but it possibly removes confusion as to whether we're dealing with a Verb (do something) or a Substantive (a thing).

So, likewise, I would put...

* "Mehrere exportieren"
* "Diesen Tiddler exportieren"
* etc...